server.js

```diff
@@ -54,6 +54,7 @@ app.use(function (req, res, next) {
         rootURL,
         (webSocketURL || 'wss://ws.blockchain.info'),
         (apiDomain || 'https://api.blockchain.info'),
+        'https://api.blockchain.info',
```

Comment:
You shouldn't have a duplicate entry. In `server.js` you can prevent that with e.g. `apiDomain === `https://api.blockchain.info` ? '' : 'https://api.blockchain.info'`

`server.js` is only used for local development, not for deployment, so headers need to modified for that too.